### PR TITLE
Use IPv6 in resolver cache tests

### PR DESCRIPTION
**Pull request by adambkaplan**
Use "::1/128" instead of 127.0.0.1

### CODE REVIEW

File: pkg/image/apiserver/registryhostname/environmentresolvercache_test.go [hunk 1]
```diff
@@ -57,13 +57,13 @@ func TestServiceResolverCache(t *testing.T) {
 	c = fakeRetriever{
 		service: &corev1.Service{
 			Spec: corev1.ServiceSpec{
-				ClusterIP: "127.0.0.1",
+				ClusterIP: "::1/128",
```
Comment (dmage) [inline pkg/image/apiserver/registryhostname/environmentresolvercache_test.go]: @adambkaplan why do you need `/128` here? I wouldn't expect to see CIDR here.

Comment (adambkaplan) [reply pkg/image/apiserver/registryhostname/environmentresolvercache_test.go]: @dmage you're right, should just be `::1`. Make me wonder how this is passing (perhaps the /128 is ignored?)

Comment (adambkaplan) [reply pkg/image/apiserver/registryhostname/environmentresolvercache_test.go]: @dmage fixed - looks like the function under test just accepts the string as-is, doesn't do any verification

**Pull request merged**